GoogleDataTransport/GDTLibrary/Public/GDTConsoleLogger.h

```diff
@@ -56,7 +56,12 @@ typedef NS_ENUM(NSInteger, GDTMessageCode) {
   GDTMCETransportBytesError = 1005,
 
   /** For general purpose error messages in a dependency. */
-  GDTMCEGeneralError = 1006
+  GDTMCEGeneralError = 1006,
+
+  /** For fatal errors. Please go to https://github.com/firebase/firebase-ios-sdk/issues and open
+   * an issue if you encounter an error with this code.
+   */
+  GDTMCEFatalAssertion = 1007
```

Comment:
I'd prefer something like "Serious" over "Fatal".  From an app developer perspective, It won't be fatal if GDT fails while doing CoreDiagnostics. 

I think it should be left to the clients to determine if something is fatal or not.

Reply:
This has to do with the operation of GDT, not CoreDiagnostics. Maybe while FIRCD is the only consumer this is true, but soon SDK service data will transit through GDT.

Reply:
Since the comment seems to be intended for anyone who sees a crash, will you clarify in the comment that a fatal error in GDT may not be fatal for the app depending upon what the GDT client is using it for.